specs/experimental/fault-proof/stage-one/bond-incentives.md

```diff
@@ -58,3 +66,94 @@ are located on the same level as honest claims, but situated to its right (i.e.
+Large Preimage Proposal.
+
+We use a multiplicative scaling mechanism to guarantee that the ratio between bonds remains constant.
+We determine the multiplier based on the proposed `MAX_DEPTH` of 73.
+We can use the formula `x = (200_000_000 / 400_000) ** (1 / 73)` to determine that `x = 1.089`.
+At each depth `N`, the amount of gas charged is therefore `400_000 * (1.089 ** N)`
+
+### Required Bond Formula
+
+Applying the [Gas Price Assumption](#gas-price-assumption) and [Bond Scaling](#bond-scaling) specifications, we have a
+`getRequiredBond` function:
+
+```python
+def get_required_bond(position):
+    assumed_gas_price = 200 gwei
+    base_gas_charged = 400_000
+    gas_charged = 400_000 * (1.089 ** position.depth)
+    return gas_charged * assumed_gas_price
+```
+
+### DelayedWETH
+
+FPM introduces a contract `DelayedWETH` designed to hold the bonded ETH for each
+[Fault Dispute Game](./fault-dispute-game.md).
+`DelayedWETH` is an extended version of the standard `WETH` contract that introduces a delayed unwrap mechanism that
+allows an owner address to function as a backstop in the case that a Fault Dispute Game would
+incorrectly distribute bonds.
+
+`DelayedWETH` is modified from `WETH` as follows:
+
+- `DelayedWETH` is an upgradeable proxy contract.
+- `DelayedWETH` has an `owner()` address
+- `DelayedWETH` has a `delay()` function that returns a period of time that withdrawals will be delayed.
+- `DelayedWETH` has an `unlock(guy,wad)` function that modifies a mapping called `withdrawals` keyed as
+`withdrawals[msg.sender][guy] => WithdrawalRequest` where `WithdrawalRequest` is
+`struct Withdrawal Request { uint256 amount, uint256 timestamp }`. When `unlock` is called, the timestamp for
+`withdrawals[msg.sender][guy]` is set to the current timestamp and the amount is increased by the given amount.
+`unlock(guy,wad)` is not callable when `SuperchainConfig.paused()` is `true`.
+- `DelayedWETH` modifies the `WETH.withdraw` function such that an address *must* provide a "sub-account" to withdraw
+from. The function signature becomes `withdraw(guy,wad)`. The function retrieves `withdrawals[msg.sender][guy]` and
+checks that the current `block.timestamp` is greater than the timestamp on the withdrawal request plus the `delay()`
+seconds and reverts if not. It also confirms that the amount being withdrawn is less than the amount in the withdrawal
+request. Before completing the withdrawal, it reduces the amount contained within the withdrawal request. The original
+`withdraw(wad)` function becomes an alias for `withdraw(msg.sender, wad)`. `withdraw(guy,wad)` will not be callable
+when `SuperchainConfig.paused()` is `true`.
+- `DelayedWETH` has a `hold()` function that allows the `owner()` address to give itself an allowance from any address.
+- `DelayedWETH` has a `recover()` function that allows the `owner()` address to recover any amount of ETH from the
+contract.
+
```

Comment:
A mermaid diagram would be super helpful here to describe its interactions with the rest of the system